share/picongpu/tests/Synchrotron_plugin/lib/validate.py

```diff
@@ -7,15 +7,8 @@
 # Used when comparing two sets of histogram data with different binning
 # given two set of vectors x0, y0 and x1, y1 we interpolate second set to the first set and subtract them
 # interpolate y1 to x0 and subtract y0 from y1
+# returns x0 and relative error of y1 compared to y0
 def subtract_functions(x0, y0, x1, y1):
-    minX = max(np.min(x0), np.min(x1))
-    maxX = min(np.max(x0), np.max(x1))
-    #
-    x0 = [x for x in x0 if x >= minX and x <= maxX]
-    x1 = [x for x in x1 if x >= minX and x <= maxX]
-    y0 = [y for x, y in zip(x0, y0) if x >= minX and x <= maxX]
-    y1 = [y for x, y in zip(x1, y1) if x >= minX and x <= maxX]
-
     y1 = np.interp(x0, x1, y1)  # interpolate y1 to the x0 values
```

Comment:
Okay, so you are saying that you are fine with the default behaviour of `np.interp` here? I would have assumed that for histograms using `left=0, right=0` would have been the most faithful extrapolation.

Reply:
You are right. I did that. I also changed the name to be more descriptive